File: Level 1/Week01/Day1/hhs/1.java
class Solution {
    public boolean isPrimeNumber(int n) {
        for (int i = 2; i * i <= n; i++) {
            if (n % i == 0) return false;
        }

        return true;
    }

    public int solution(int n) {
        int answer = 0;

        for (int i = 2; i <= n; i++) {
            if (isPrimeNumber(i)) answer++;
        }

        return answer;
    }
}

File: Level 2/Week04/Day27/hhs/1.kt
class Solution {
    fun solution(n: Int, a: Int, b: Int): Int {
        var answer = 0

        var _a = a
        var _b = b

        while (_a != _b) {
            _a = _a / 2 + _a % 2
            _b = _b / 2 + _b % 2

            answer++
        }

        return answer
    }
}